lib/src/marker_cluster_layer.dart

```diff
@@ -514,7 +515,7 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
     return layers;
   }
 
-  List<Widget> _buildLayers() {
+  List<Widget?> _buildLayers() {
```

Comment:
I'm starting to test this locally with null-safety enabled now. I get an error since this should be List<Widget>, same goes for line 527 below. It means that line 529 has to be modified as well to either:

```dart
if (_polygon != null) layers.add(_polygon!);
```

(note the added ! on _polygon)

or

```dart
final currentPolygon = _polygon;
if (currentPolygon != null) layers.add(currentPolygon);
```

Reply:
I think the update from nullsafety.3 which I merged into nullsafety takes care of this, but making Widget non-nullable again, but lmk